tests/route/test_route_flap.py

```diff
@@ -152,16 +158,30 @@ def get_exabgp_port(duthost, tbinfo, dev_port):
     tor1_exabgp_port = EXABGP_BASE_PORT + tor1_offset
     return tor1_exabgp_port
 
+
 def is_dualtor(tbinfo):
     """Check if the testbed is dualtor."""
     return "dualtor" in tbinfo["topo"]["name"]
 
-def test_route_flap(duthost, tbinfo, ptfhost, ptfadapter,
-                    get_function_conpleteness_level, announce_default_routes):
+def has_external_route(asichost, route_prefix):
```

Comment:
Should this function be changed as well, given that the latest commit removes get_internal_bgp_peers_for_chassis_per_asic() ?